src/components/AppFooter.vue

```diff
@@ -3,7 +3,9 @@
     <hr class="my-1" />
     <div class="footer-section">
       <span>&copy; Verida {{ currentYear }}</span>
-      <a href="##">Terms & Conditions</a>
+      <a href="https://www.verida.io/terms-and-conditions" target="_blanck"
```

Comment:
There's a typo, should be `target="_blank"`
But do we need to open it in a new tab in the first place?

Reply:
Thanks will fix that and also add the `rel="noopener"` attribute to ensure the link doesn't grant the new browsing context access to the document that opened it see [link](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener)


> But do we need to open it in a new tab in the first place?
yes, we have been to open a new browser because users might want to view their account profile before navigating to a different tab, do you think it's not needed?

Reply:
There's a debate on this UX topic.
Check this article for instance https://css-tricks.com/use-target_blank/